oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,146 @@
+===========================
+OEP-54: Core Contributors
+===========================
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-54 <oep-0054-core-contributors>`
+   * - Title
+     - Core Contributor Program
+   * - Last Modified
+     - 2021-12-20
+   * - Authors
+     - Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Xavier Antoviaque <xavier@opencraft.com>
+   * - Status
+     - Draft
+   * - Type
+     - Process
+   * - Created
+     - 2021-12-20
+   * - Review Period
+     - TBD
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Abstract
+========
+
+This document defines the Core Contributor (CC) Program, a program that grants rights to individuals
+in the Open edX community that allow them to actively participate in defining and deciding the
+direction of the platform.
+
+Motivation
+==========
+
+The Open edX project benefits from a wide variety of perspectives at all levels. Having an inclusive
+group of people participating in the project and helping to shape its direction will lead to a more
+vibrant platform ecosystem that fosters stronger innovative collaboration within the Open edX community,
+increased velocity that accelerates platform advancement, and increased capacity to own and maintain
+all aspects of the platform.
+
+Specification
+=============
+
+Core Contributor Roles
+----------------------
+
+There are many contributors to the Open edX platform that participate in a variety of ways - writing
+code for the Open edX software, translating the project, performing Quality Assurance tasks, participating
+in the Marketing working group, writing documentation, and more. At the time of writing we have identified
+the following roles: Project Manager, Translation Lead, Code Contributors, Forum Moderators, UI/UX Designers,
+Marketing Team Members, Documentation Writers, QA, and Product Managers.
+
+
+All CCs are granted some Rights to the project as part of participating - the ability to
+merge code, the final say on marketing copy, etc. In exchange for these Rights to contribute, we ask that
+CCs abide by their Responsibilities. For more on the CC roles, and Rights & Responsibilities, see:
+
+- `Core Contributor Role Definitions <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2759460357/Core+Contributor+Role+Definitions>`_
+- `Core Contributor Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2952003698/Core+Contributor+Rights+Responsibilities+excluding+code+contributors>`_
+- `Code Committer Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Rights+Responsibilities+for+Code+Contributors>`_
```

Comment:
They are core, but I don't want them defined in two places - that makes multiple places to update, high chance of getting out of sync and unclear which document is the main one. I defaulted to the wiki as the main branch because it's much easier to update the wiki (particularly, by those who don't code).